contributing-guides/style-guide.de.md

```diff
@@ -0,0 +1,92 @@
+# Style guide
+
+ Diese Seite listet alle Regeln für `tldr`-Seiten auf.
+
+ ## Layout
+
+ Eine Standard-`tldr`-Seite sollte dem folgenden Format entsprechen:
+
+ ```
+ # kommando-name
+
+ > Kurze Beschreibung.
+ > Möglichst nur eine; wenn nötig, sind zwei akzeptabel.
+ > More information: <https://example.com>.
+
+ - Beispielbeschreibung:
+
+ `kommando -opt1 -opt2 -arg1 {{arg_wert}}`
+
+ - Beispielbeschreibung:
+
+ `kommando -opt1 -opt2`
+ ```
+
+ Es gibt einen Linter, der das obige Format prüft.
+ Er wird automatisch bei jedem Pull Request ausgeführt,
+ er kann aber auch manuell installiert werden, um seine Seiten schon vorher zu überprüfen:
+
+ ```
+ npm install tldr-lint
+ tldrl -f {{seite.md}}
+ ```
+
+ Für andere Optionen von `tldrl`, wie linten eines ganzen Ordners:
+ [`tldr tldrl`](https://github.com/tldr-pages/tldr/blob/main/pages/common/tldrl.md)
+
+ Viele Clients unterstützen die `--render` Flag zum Anzeigen einer Seite:
+
+ ```
+ tldr --render {{page.md}}
+ ```
+
+ ## Token-Syntax
+
+ Eingaben der Nutzer\*innen sollten die `{{Token}}`-Syntax nutzen,
+ damit `tldr`-Clients sie hervorheben können.
+
+ Die folgenden Regeln sollten für Tokens beachtet werden:
+
+ 1. Kurze und deskriptive Tokens,
+    z. B. `{{source_file}}` oder `{{wallet.txt}}`.
+ 2. Nutze `snake_case` <!--TODO: german wikipedia article for snake_case--> für Tokens, die aus mehreren Wörtern bestehen.
+ 3. Nutze `{{filename}}` statt `{{file_name}}`.
+ 4. Nutze für Referenzen auf Pfade von Dateien oder Ordnern das Format `{{path/to/<Platzhalter>}}`.
+    Beispielsweise `ln -s {{path/to/file}} {{path/to/symlink}}`.
+    Nutze für eine Lücke, in die Dateien und Ordner passen `{{path/to/file_or_directory}}`
+ 5. Folge der `{{path/to/<Platzhalter>}}`-Konvention für alle Pfad-bezogenen Kommandos, außer wenn der
+    Ort der Datei implizit ist.
+ 6. Wenn ein Kommando eine bestimmte Dateiendung erwartet, nutze sie.
+    Beispiel: `unrar x {{compressed.rar}}`.
+    Für eine generelle Dateiendung, nutze `{{.ext}}`, aber **nur**, wenn eine Endung wirklich nötig ist.
+    Beispielsweise, in find.md's Beispiel "Find files by extension" (`find {{root_path}} -name '{{*.ext}}'`)
+    erklärt `{{*.ext}}` den Befehl ohne unnötig spezifisch zu sein;
+    Aber in einem Befehl wie `wc -l {{file}}`, genügt `{{file}}` (ohne Endung).
+ 7. Wenn das Beispiel mit einem konkreten Wert klarer ist, nutze einen Beispielwert.
+    Nutze beispielsweise `iostat {{2}}` statt `iostat {{interval_in_secs}}`.
+ 8. Wenn ein Kommando irreversible Änderungen am Dateisystem oder Geräten vornimmt, schreibe jedes Beispiel so, dass es nicht blind copy-pastet werden kann.
+    Schreibe beispielsweise statt `ddrescue --force --no-scrape /dev/sda /dev/sdb` `ddrescue --force --no-scrape {{/dev/sdX}} {{/dev/sdY}}` und nutze den `{{/dev/sdXY}}`-Platzhalter für *block devices* statt `/dev/sda1`.
```

Comment:
```suggestion
    Schreibe beispielsweise `ddrescue --force --no-scrape {{/dev/sdX}} {{/dev/sdY}}` statt  `ddrescue --force --no-scrape /dev/sda /dev/sdb`und nutze den `{{/dev/sdXY}}`-Platzhalter statt `/dev/sda1` für *blockgeräte*.
```
This looks a bit confusing, but I just rearanged the sentense a bit.

Reply:
blockgeräte? seriously?

Reply:
I don't see a problem with it
![image](https://user-images.githubusercontent.com/50295997/129370012-17ff13df-2346-483b-9605-7f21c8155246.png)

Reply:
It's still horrible.